x2i-data/apta.yaml

```diff
@@ -0,0 +1,993 @@
+---
+name: APTA
+help: 'https://web.archive.org/web/20230407205814/https://docs.google.com/spreadsheets/u/0/d/1vrO-PypNiL8SlYCGEmuVFfHdQl2daXkM9tXkek0iUNQ/htmlview'

```

Comment:
If the google docs link is still active, it's fine to use that instead of an archive.org link. I only used it for Z-SAMPA because the original page for that one is down.

Reply:
I archived it myself just in case the author deletes the spreadsheet (as it's often the case with obscure old gsuite files I come across). In case the link goes down, the archive link can be this one.

Reply:
Maybe we can mirror it as an xlsx file in the resources folder with an attribution to the original source?

Reply:
I'd rather use some format that Github can render. I'm honestly not sure whether it can preview Excel sheets.